styles/style.css

```diff
@@ -0,0 +1,111 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@200;400&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Inter", sans-serif, "Times New Roman", Times, serif;
+  color: #172b4d;
+  text-align: center;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.green {
+  color: #36b37e;
+}
+
+a {
+  text-decoration: none;
+}
+
+#container {
+  width: 375px;
+}
+
+.btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  background-color: transparent;
+  border: transparent;
+  cursor: pointer;
+}
+
+/* header start */
+#toolbar_wrapper {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 48px;
+  background-color: #fff;
+  z-index: 10;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 89.33333%;
+  margin: 0 auto;
+}
```

Comment:
- [ ] Kindly follow the given design on the mobile screen and fix the toolbar styling. You should follow the typography, font size, weight, color, margin, and padding as same as the design template.
Please take a look at the two images down below.
### **Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/120844235-65821e00-c56f-11eb-9199-8d52ff0cce6d.png)

### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)**
![Screenshot1](https://user-images.githubusercontent.com/47625407/120844179-54d1a800-c56f-11eb-9e08-da8ff8beb9ef.png)